### PR TITLE
Ensembl REST API wrappers

### PR DESCRIPTION
**Pull request by anilthanki**
Pull request for Ensembl REST wrappers

### CODE REVIEW

File: tools/Ensembl-REST/get_gene/get_gene_info.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<tool id="get_gene_info_wrapper" name="get_gene_info_from_Ensembl" version="0.1.1">
```
Comment (nsoranzo) [inline tools/Ensembl-REST/get_gene/get_gene_info.xml]: `name` is free text, I'd use `"Get gene info from Ensembl"`. And just `using REST API` in `<description>`.

File: tools/Ensembl-REST/get_gene/get_gene_info.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            -e $expand
+            -u $utr
+            -f $format_selector
+            -i $input
+            > $output
+        ]]>
+    </command>
+
+    <inputs>
+    <param name="input" type="data" format="txt" label="Ensembl ID"/>
```
Comment (nsoranzo) [inline tools/Ensembl-REST/get_gene/get_gene_info.xml]: s/ID/IDs/

**Comment by bgruening** [comment]
Related: https://github.com/bgruening/galaxytools/tree/master/tools/uniprot_id_mapping (uniprot)
https://github.com/bgruening/galaxytools/tree/master/tools/rest_tool (pubchem)

Maybe we should create a suite out of it?
Great work @anilthanki!

File: tools/Ensembl-REST/get_gene/get_gene_info.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  
+    <param name="expand" type="boolean" truevalue="1" falsevalue="0" label="Expand" help="Expands the search to include any connected features. e.g. If the object is a gene, its transcripts, translations and exons will be returned as well." />
+  
+    <param name="utr" type="boolean" truevalue="1" falsevalue="0" label="UTR" help="Include 5' and 3' UTR features. Only available if the expand option is used." />
```
Comment (nsoranzo) [inline tools/Ensembl-REST/get_gene/get_gene_info.xml]: There is no `utr` parameter in the docs: http://rest.ensembl.org/documentation/info/lookup_post . Is this a copy-paste error?

**Pull request merged**